model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -627,6 +627,9 @@ message StateKey {
   message Runner {
     // (Required) Opaque information supplied by the runner. Used to support
     // remote references.
+    //
+    // Used by state backed iterable. And in this use case, request type can
```

Comment:
Add a link to the Fn API remote references documentation in the statement above.

Also, add a link to the documentation about state backed iterables.